### PR TITLE
Remove syntax (WIP)

### PR DESCRIPTION
**Pull request by danieljharvey**

### CODE REVIEW

File: src/Language/JavaScript/Pretty/Printer.hs [hunk 1]
```diff
@@ -8,20 +10,21 @@ module Language.JavaScript.Pretty.Printer
...
 #endif
-import Data.Text.Lazy (Text)
-import Language.JavaScript.Parser.AST
-import Language.JavaScript.Parser.SrcLocation
-import Language.JavaScript.Parser.Token
-import qualified Blaze.ByteString.Builder.Char.Utf8 as BS
-import qualified Data.ByteString.Lazy as LB
-import qualified Data.Text.Lazy.Encoding as LT
-import qualified Codec.Binary.UTF8.String as US
+import qualified Blaze.ByteString.Builder.Char.Utf8     as BS
```
Comment (erikd) [inline src/Language/JavaScript/Pretty/Printer.hs:20]: Syntacically null whitespace only changes will never be accepted.

File: src/Language/JavaScript/Pretty/Printer.hs [hunk 2]
```diff
@@ -149,30 +146,30 @@ instance RenderJS [JSExpression] where
...
-    (|>) pacc (JSBinOpNeq        annot)  = pacc |> annot |> "!="
-    (|>) pacc (JSBinOpOf         annot)  = pacc |> annot |> "of"
-    (|>) pacc (JSBinOpOr         annot)  = pacc |> annot |> "||"
-    (|>) pacc (JSBinOpPlus       annot)  = pacc |> annot |> "+"
-    (|>) pacc (JSBinOpRsh        annot)  = pacc |> annot |> ">>"
-    (|>) pacc (JSBinOpStrictEq   annot)  = pacc |> annot |> "==="
-    (|>) pacc (JSBinOpStrictNeq  annot)  = pacc |> annot |> "!=="
-    (|>) pacc (JSBinOpTimes      annot)  = pacc |> annot |> "*"
-    (|>) pacc (JSBinOpUrsh       annot)  = pacc |> annot |> ">>>"
+    (|>) pacc (JSBinOpAnd        annot) = pacc |> annot |> "&&"
```
Comment (erikd) [inline src/Language/JavaScript/Pretty/Printer.hs:146]: That change *might* be worthwhile, but only as a "whitespace only" commit.

**Review by erikd — CHANGES_REQUESTED** [review]
Sorry, this is nowhere near ready to be PRed.

**Comment by danieljharvey** [comment]
Hello! Very sorry! I am playing with a fork of this to try out parsing a subset of JS syntax, I meant to make a PR against my own fork, and did it on here by mistake. Sorry for wasting your time, thanks for all your work on this library.

**Pull request closed without merging**